src/main.cpp

```diff
@@ -228,6 +235,8 @@ void setup() {
     logMemory("CONF");
     battery.setUpdateCallbacks(new MyBatteryUpdateCallbacks());
     battery.init(cfg.devmode);
+    battery.update();
+    // powerInit();
```

Comment:
We have this issue, when try to validate the low voltage condition, the T7 device for example, go to panic @roberbike

Reply:
Yes ,it is true. A solar version without battery voltage control does not make much sense. ADC for it is neecesary, with resistors divider or other type, I think.